custom/src/main/java/com/splunk/opentelemetry/SplunkConfiguration.java

```diff
@@ -187,6 +189,16 @@ Map<String, String> customize(ConfigProperties config) {
       customized.put("otel.exporter.otlp.headers", otlpHeaders);
     }
 
+    if (config.getBoolean(SPLUNK_AUTO_DETECT_SERVICE_NAME, true)) {
+      if (ServiceNameDetector.serviceNameNotConfigured(config)) {
+        String serviceName = ServiceNameDetector.detectServiceName();
+        if (serviceName != null) {
+          log.info("Auto-detected service name '{}'.", serviceName);
```

Comment:
should we also log some kind of description (path to the web.xml?) based on what we auto-detected?

Reply:
If it's not too much trouble, then why not -- it'd make our lives easier. I'd log it at debug though, some customers are kinda sensitive to the amount of logs printed by the agent.